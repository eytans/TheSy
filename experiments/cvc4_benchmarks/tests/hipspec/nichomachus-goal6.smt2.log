19:39:31 [WARN] Collecting statistics
19:39:31 [WARN]  ex is: zero
19:39:31 [WARN]  ex is: (succ zero)
19:39:31 [WARN]  ex is: (succ (succ zero))
19:39:31 [WARN] Running exploration without proof mode, but goals were given
19:39:31 [WARN] Proof mode: true
19:39:31 [WARN]  ex is: zero
19:39:31 [WARN]  ex is: (succ zero)
19:39:31 [WARN]  ex is: (succ (succ zero))
19:39:31 [WARN] Running exploration without proof mode, but goals were given
19:39:31 [WARN] Proof mode: true
19:39:31 [WARN] proved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:39:31 [WARN] proved: (mult ?ts_ph_Nat_0 zero) => zero
19:39:31 [WARN] proved: (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:39:31 [WARN] proved: (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:39:31 [WARN] proved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0) ?ts_ph_Nat_1)
19:39:31 [WARN] proved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0) ?ts_ph_Nat_1) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:39:31 [WARN] proved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_0)) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_0)
19:39:31 [WARN] proved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_0) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_0))
19:39:31 [WARN] proved: (mult (succ ?ts_ph_Nat_0) (succ zero)) => (succ ?ts_ph_Nat_0)
19:39:31 [WARN] proved: (succ ?ts_ph_Nat_0) => (mult (succ ?ts_ph_Nat_0) (succ zero))
19:39:31 [WARN] proved: (plus (succ ?ts_ph_Nat_0) (tri ?ts_ph_Nat_0)) => (tri (succ ?ts_ph_Nat_0))
19:39:31 [WARN] proved: (tri (succ ?ts_ph_Nat_0)) => (plus (succ ?ts_ph_Nat_0) (tri ?ts_ph_Nat_0))
19:39:33 [WARN] proved: (mult ?ts_ph_Nat_0 ?ts_ph_Nat_0) => (mult (mult ?ts_ph_Nat_0 ?ts_ph_Nat_0) (succ zero))
19:39:33 [WARN] proved: (mult (mult ?ts_ph_Nat_0 ?ts_ph_Nat_0) (succ zero)) => (mult ?ts_ph_Nat_0 ?ts_ph_Nat_0)
19:39:33 [WARN] proved: (plus ?ts_ph_Nat_1 ?ts_ph_Nat_0) => (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)
19:39:33 [WARN] proved: (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) => (plus ?ts_ph_Nat_1 ?ts_ph_Nat_0)
19:39:33 [WARN] proved: (mult ?ts_ph_Nat_0 (succ zero)) => ?ts_ph_Nat_0
19:39:33 [WARN] proved: (mult (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0) ?ts_ph_Nat_1) => (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1))
19:39:33 [WARN] proved: (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1)) => (mult (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0) ?ts_ph_Nat_1)
19:39:34 [WARN] proved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2)) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2)))
19:39:34 [WARN] proved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2))) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2))
19:39:38 [WARN] proved: (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1)) => (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:39:38 [WARN] proved: (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1))
