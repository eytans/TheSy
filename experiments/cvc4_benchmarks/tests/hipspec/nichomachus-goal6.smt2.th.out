19:39:31 [0m[33m[WARN] [0mCollecting statistics
(plus zero ?n) => ?n
(plus (succ ?n) ?m) => (succ (plus ?n ?m))
(succ (plus ?n ?m)) => (plus (succ ?n) ?m)
(mult zero ?n) => zero
(mult (succ ?n) ?m) => (plus (mult ?n ?m) ?m)
(plus (mult ?n ?m) ?m) => (mult (succ ?n) ?m)
(tri zero) => zero
zero => (tri zero)
(tri (succ ?n)) => (plus (tri ?n) (succ ?n))
(plus (tri ?n) (succ ?n)) => (tri (succ ?n))
(cubes zero) => zero
zero => (cubes zero)
(cubes (succ ?n)) => (plus (cubes ?n) (mult (succ ?n) (mult (succ ?n) (succ ?n))))
(plus (cubes ?n) (mult (succ ?n) (mult (succ ?n) (succ ?n)))) => (cubes (succ ?n))
19:39:31 [0m[33m[WARN] [0m ex is: zero
19:39:31 [0m[33m[WARN] [0m ex is: (succ zero)
19:39:31 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:39:31 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:39:31 [0m[33m[WARN] [0mProof mode: true
19:39:31 [0m[33m[WARN] [0m ex is: zero
19:39:31 [0m[33m[WARN] [0m ex is: (succ zero)
19:39:31 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:39:31 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:39:31 [0m[33m[WARN] [0mProof mode: true
19:39:31 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:39:31 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_0 zero) => zero
19:39:31 [0m[33m[WARN] [0mproved: (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:39:31 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:39:31 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0) ?ts_ph_Nat_1)
19:39:31 [0m[33m[WARN] [0mproved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0) ?ts_ph_Nat_1) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:39:31 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_0)) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_0)
19:39:31 [0m[33m[WARN] [0mproved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_0) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_0))
19:39:31 [0m[33m[WARN] [0mproved: (mult (succ ?ts_ph_Nat_0) (succ zero)) => (succ ?ts_ph_Nat_0)
19:39:31 [0m[33m[WARN] [0mproved: (succ ?ts_ph_Nat_0) => (mult (succ ?ts_ph_Nat_0) (succ zero))
19:39:31 [0m[33m[WARN] [0mproved: (plus (succ ?ts_ph_Nat_0) (tri ?ts_ph_Nat_0)) => (tri (succ ?ts_ph_Nat_0))
19:39:31 [0m[33m[WARN] [0mproved: (tri (succ ?ts_ph_Nat_0)) => (plus (succ ?ts_ph_Nat_0) (tri ?ts_ph_Nat_0))
19:39:33 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_0 ?ts_ph_Nat_0) => (mult (mult ?ts_ph_Nat_0 ?ts_ph_Nat_0) (succ zero))
19:39:33 [0m[33m[WARN] [0mproved: (mult (mult ?ts_ph_Nat_0 ?ts_ph_Nat_0) (succ zero)) => (mult ?ts_ph_Nat_0 ?ts_ph_Nat_0)
19:39:33 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_1 ?ts_ph_Nat_0) => (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)
19:39:33 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) => (plus ?ts_ph_Nat_1 ?ts_ph_Nat_0)
19:39:33 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_0 (succ zero)) => ?ts_ph_Nat_0
19:39:33 [0m[33m[WARN] [0mproved: (mult (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0) ?ts_ph_Nat_1) => (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1))
19:39:33 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1)) => (mult (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0) ?ts_ph_Nat_1)
19:39:34 [0m[33m[WARN] [0mproved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2)) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2)))
19:39:34 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2))) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2))
19:39:38 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1)) => (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:39:38 [0m[33m[WARN] [0mproved: (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1))
done in 39996
